deployment/README.md

```diff
@@ -63,83 +102,115 @@ The deployer must run under a service account defined as follows:
     - name: logging-deployer
     API
 
-    oc policy add-role-to-user edit \
+    $ oc policy add-role-to-user edit \
               system:serviceaccount:logging:logging-deployer
 
 Note: change `:logging:` above to match the project name.
 
+The policy manipulation is required in order for the deployer pod to
+create secrets, templates, and deployments in the project. By default
+service accounts are not allowed to do this.
+
+The fluentd deployment also requires a service account which the deployer
+will create that must be given special privileges. Edit the privileged
+security configuration with the following:
+
+    oc edit scc/privileged
+
+Add one line as the user at the end: (note, change `:logging:` below to the project of your choice)
+
+    - system:serviceaccount:logging:aggregated-logging-fluentd
+
+Give the account access to read labels from all pods (again with the correct project):
+
+    openshift admin policy add-cluster-role-to-user cluster-reader system:serviceaccount:logging:aggregated-logging-fluentd
```

Comment:
I had a reason to prefer "openshift admin" but I don't think it applies any more.